Sources/UIKitExtensions/UISearchBar+FunctionConfigurationSet.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  UISearchBarFunctions.swift
+//  ViewConfigurator-iOS
+//
+//  Created by Andrii Pazynych on 27.03.18.
+//  Copyright © 2018 ImagineOn GmbH. All rights reserved.
+//
+
+
+import UIKit
+public extension ConfigurationSet where Base: UISearchBar {
+    
+    @available(iOS 3.0, *)
+    func setShowsCancelButton(_ showsCancelButton: Bool, animated: Bool) -> ConfigurationSet<Base> {
+        return set { ( UISearchBar: Base ) in
+            UISearchBar.setShowsCancelButton(showsCancelButton, animated: animated)
```

Comment:
This function is only useful for existing UISearchBar whose showsCancelButton change should be animated.
The showsCancelButton Property ist writable directly so there is no need for this function.